set.mm

```diff
@@ -469438,6 +469469,445 @@ Splicing words (substring replacement)
+       B /\ A. x e. A A. y e. A ( x .<_ y -> ( F ` x ) .c_ ( F ` y ) ) ) ) ) $=
+      ( vf wcel wa cfv wral cmnt co cmap cv wbr wi wf crab mntoval eleq2d fveq1
+      wceq breq12d imbi2d 2ralbidv elrab syl6bb cbs fvexi elmap anbi1i ) GIQHJQ
+      RZEGHUAUBZQZEDCUCUBZQZAUDZBUDZFUEZVGESZVHESZKUEZUFZBCTACTZRZCDEUGZVNRVBVD
+      EVIVGPUDZSZVHVQSZKUEZUFZBCTACTZPVEUHZQVOVBVCWCEABCDPFGHIJKLMNOUIUJWBVNPEV
+      EVQEULZWAVMABCCWDVTVLVIWDVRVJVSVKKVGVQEUKVHVQEUKUMUNUOUPUQVFVPVNDCEDHURMU
+      SCGURLUSUTVAUQ $.
+  $}
+
+  ${
+    $d .<_ x y $.  $d .c_ x y $.  $d A x y $.  $d F x y $.  $d V x y $.
+    $d W x y $.  $d X x y $.  $d Y y $.  $d ph x $.
+    ismntd.1 $e |- A = ( Base ` V ) $.
+    ismntd.2 $e |- B = ( Base ` W ) $.
+    ismntd.3 $e |- .<_ = ( le ` V ) $.
+    ismntd.4 $e |- .c_ = ( le ` W ) $.
+    ismntd.5 $e |- ( ph -> V e. C ) $.
+    ismntd.6 $e |- ( ph -> W e. D ) $.
+    ismntd.7 $e |- ( ph -> F e. ( V Monot W ) ) $.
+    ismntd.8 $e |- ( ph -> X e. A ) $.
+    ismntd.9 $e |- ( ph -> Y e. A ) $.
+    ismntd.10 $e |- ( ph -> X .<_ Y ) $.
+    $( Property of being a monotone increasing function, deduction version.
+       (Contributed by Thierry Arnoux, 24-Apr-2024.) $)
+    ismntd $p |- ( ph -> ( F ` X ) .c_ ( F ` Y ) ) $=
+      ( vx vy cv wbr cfv wi wral wcel co w3a wf wa ismnt biimp3a simprd syl3anc
+      cmnt wceq breq1 fveq2 breq1d imbi12d breq2 breq2d eqidd rspc2vd mp2d ) AU
+      CUEZUDUEZGUFZVJFUGZVKFUGZLUFZUHZUDBUIUCBUIZJKGUFZJFUGZKFUGZLUFZAHDUJZIEUJ
+      ZFHIUSUKUJZVQQRSWBWCWDULBCFUMZVQWBWCWDWEVQUNUCUDBCFGHIDELMNOPUOUPUQURUBAV
+      RWAUHJVKGUFZVSVNLUFZUHVPUCUDJKBBBVJJUTZVLWFVOWGVJJVKGVAWHVMVSVNLVJJFVBVCV
+      DVKKUTZWFVRWGWAVKKJGVEWIVNVTVSLVKKFVBVFVDTAWHUNBVGUAVHVI $.
+  $}
+
+  ${
+    $d A x y $.  $d F x y $.  $d V x y $.  $d W x y $.
+    mntf.1 $e |- A = ( Base ` V ) $.
+    mntf.2 $e |- B = ( Base ` W ) $.
+    $( A monotone function is a function.  (Contributed by Thierry Arnoux,
+       24-Apr-2024.) $)
+    mntf $p |- ( ( V e. X /\ W e. Y /\ F e. ( V Monot W ) ) -> F : A --> B ) $=
+      ( vx vy wcel cmnt co cv cple cfv wbr wral eqid w3a wf wi wa ismnt biimp3a
+      simpld ) DFLZEGLZCDEMNLZUAABCUBZJOZKOZDPQZRULCQUMCQEPQZRUCKASJASZUHUIUJUK
+      UPUDJKABCUNDEFGUOHIUNTUOTUEUFUG $.
+  $}
+
+  ${
+    $d .<_ a b v w $.  $d .c_ a b v w $.  $d A a b f g v w x y $.
+    $d B a b f g v w x y $.  $d V a b f g v w x y $.  $d W a b f g v w x y $.
+    $d X a b f g v w x y $.  $d Y a b f g v w x y $.
+    mgcoval.1 $e |- A = ( Base ` V ) $.
+    mgcoval.2 $e |- B = ( Base ` W ) $.
+    mgcoval.3 $e |- .<_ = ( le ` V ) $.
+    mgcoval.4 $e |- .c_ = ( le ` W ) $.
+    $( Operation value of the monotone Galois connection.  (Contributed by
+       Thierry Arnoux, 23-Apr-2024.) $)
+    mgcoval $p |- ( ( V e. X /\ W e. Y ) -> ( V MGalConn W ) = { <. f , g >. |
+       ( ( f e. ( B ^m A ) /\ g e. ( A ^m B ) ) /\ A. x e. A A. y e. B ( ( f `
+       x ) .c_ y <-> x .<_ ( g ` y ) ) ) } ) $=
```

Comment:
So we have Galois connections being "pairs", not quadruples as in the referenced article. The elements of the pairs, however, are usually extensible structures, consisting of a base set and an ordering relation (at least), so we have our for elements. Maybe this should be explaind somewhere.

Reply:
Technically this is an operation taking a pair of structures `V` and `W` (expected to be posets), which gives a relation between pairs of functions `f` and `g`.

But if such a pair of functions exist, it is unique, and the functions are bijective, so in the end indeed, the correspondance is between the two posets `V` and `W`.

I've tried to improve the comments, but I'm still missing the "unicity" and "bijective" proofs.

Reply:
I would like to have more explanations win the comments (here or for the definition), maybe also providing a link to the referenced paper. But this can be done in a separate PR.